post-processor/googlecompute-import/post-processor.go

```diff
@@ -56,13 +56,15 @@ type Config struct {
 	//may use user variables and template functions in this field. Defaults to
 	//`packer-import-{{timestamp}}.tar.gz`.
 	GCSObjectName string `mapstructure:"gcs_object_name"`
+	// Specifies the architecture or processor type that this image can support. Must be one of: `arm64` or `x86_64`. Defaults to `ARCHITECTURE_UNSPECIFIED`.
+	ImageArchitecture string `mapstructure:"image_architecture"`
 	//The description of the resulting image.
 	ImageDescription string `mapstructure:"image_description"`
 	//The name of the image family to which the resulting image belongs.
 	ImageFamily string `mapstructure:"image_family"`
 	//A list of features to enable on the guest operating system. Applicable only for bootable images. Valid
 	//values are `MULTI_IP_SUBNET`, `UEFI_COMPATIBLE`,
-	//`VIRTIO_SCSI_MULTIQUEUE` and `WINDOWS` currently.
+	//`VIRTIO_SCSI_MULTIQUEUE`, `GVNIC` and `WINDOWS` currently.
```

Comment:
I think this list increased since I made this PR. If you look at the list here https://cloud.google.com/compute/docs/reference/rest/v1/machineImages
vs https://cloud.google.com/compute/docs/images/create-custom#guest-os-features
there are quite a few that appear to be unexposed yet. I can add support for everything in the api docs, but I think that needs a go module update

Reply:
Same here, I based my reasoning on the docs for the Go API client:

```
	// Possible values:
	//   "FEATURE_TYPE_UNSPECIFIED"
	//   "GVNIC"
	//   "MULTI_IP_SUBNET"
	//   "SECURE_BOOT"
	//   "SEV_CAPABLE"
	//   "SEV_SNP_CAPABLE"
	//   "UEFI_COMPATIBLE"
	//   "VIRTIO_SCSI_MULTIQUEUE"
	//   "WINDOWS"
```

I don't know which among those values are effectively supported. Although, this is unrelated to the change at hand regarding architectures, so I think it can be left out of this PR, and we can revisit it later, what do you think?

Reply:
GVNIC is the only one directly related here, since it's an option for arm architectures. I can do the rest in a separate PR